### PR TITLE
Newsletter 260 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2023-07-19-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## Nouvelles
+
+_Aucune actualité significative n'a été trouvée cette semaine dans les listes de diffusion Bitcoin-Dev ou Lightning-Dev._
+
+## En attente de confirmation #10 : Impliquez-vous
+
+_La dernière entrée de notre [série][policy series] hebdomadaire limitée  sur le relais des transactions, l'inclusion dans la mempool et la sélection des transactions minières - y compris pourquoi Bitcoin Core a une politique plus restrictive que celle autorisée par consensus et comment les portefeuilles peuvent utiliser cette politique de la manière la plus efficace._
+
+{% include specials/policy/fr/10-get-involved.md %} {% assign timestamp="2:01" %}
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-07-19-newsletter.md]:
```suggestion
{% include specials/policy/fr/10-impliquez-vous.md %} {% assign timestamp="2:01" %}
```

File: _posts/fr/2023-05-17-policy.md [hunk 2]
```diff
@@ -63,6 +63,31 @@ h2:not(:first-of-type) { margin-top: 3em; }
...
+
+{% include specials/policy/fr/09-propositions.md %}
+
+## En attente de confirmation #10 : Impliquez-vous
+
+*Originally published in [Newsletter #260](/fr/newsletters/2023/07/19/en-attente-de-confirmation-10--impliquez-vous)*
+
+{% include specials/policy/fr/10-impliquez-vous.md %}
+
+
```
Comment (bitschmidty) [inline _posts/fr/2023-05-17-policy.md]:
```suggestion
```

File: _posts/fr/newsletters/2023-07-19-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  temps variable. Un code non-constant qui fonctionne avec des clés privées
+  et des nonces peut entraîner des [attaques par canaux secondaires][topic
+  side channels]. Voir [Newsletter #246][news246 secp] pour une occasion
+  où cela aurait pu se produire et [Newsletter #251][news251 secp] pour
+  une autre occasion et une annonce selon laquelle ce type de test était
+  prévu. {% assign timestamp="1:05:33" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27411,6347,6035,7768,1032,1063,1313" %}
+[série de politiques]: /fr/blog/waiting-for-confirmation/
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-07-19-newsletter.md]:
```suggestion
[policy series]: /fr/blog/waiting-for-confirmation/
```

File: _posts/fr/2023-05-17-policy.md [hunk 4]
```diff
@@ -63,6 +63,31 @@ h2:not(:first-of-type) { margin-top: 3em; }
...
+## En attente de confirmation #9 : Propositions de politique
+
+*Originally published in [Newsletter #259](/fr/newsletters/2023/07/12/#en-attente-de-confirmation-9--propositions-de-politique)*
+
+{% include specials/policy/fr/09-propositions.md %}
+
+## En attente de confirmation #10 : Impliquez-vous
+
+*Originally published in [Newsletter #260](/fr/newsletters/2023/07/19/en-attente-de-confirmation-10--impliquez-vous)*
```
Comment (bitschmidty) [inline _posts/fr/2023-05-17-policy.md]:
```suggestion
*Originally published in [Newsletter #260](/fr/newsletters/2023/07/19/#en-attente-de-confirmation-10--impliquez-vous)*
```

Comment (Copinmalin) [reply _posts/fr/2023-05-17-policy.md]: Thank you for correcting my mistakes

Comment (bitschmidty) [reply _posts/fr/2023-05-17-policy.md]: @Copinmalin no problem. Do you want to try rebasing https://github.com/bitcoinops/bitcoinops.github.io/pull/1229 on master to pull the recent fr PRs which will fix the builds?

For reference I locally `git fetch --all` to pull the latest, then checkout your current translation branch `git checkout Newsletter-261-translate-in-french-` (for fr 261), then `git rebase origin/master` to rebase the 261 branch on the latest changes from master. There may be conflicts depending on the state of your 261 changes which you can resolve and then finish the 261 translation. Ill then use `make preview` to iterate on changes and test links locally and `make production` to run more checks like the ones run by netlify and travis here.

**Pull request merged**